### DISCUSSION TITLE
一对一一对多查询怎么进行resultmap映射

### DISCUSSION THREAD

**Discussion by cnwzhu** [rbatis/rbatis#558] [Q&A]
目前没有看到一次关联查询映射的例子，支持不支持呢

**Comment by zhuxiujia** [comment]
> 目前没有看到一次关联查询映射的例子，支持不支持呢

关联查询可以用pysql! 或者 html_sql!

**Reply by cnwzhu** [reply]
我知道怎么写这个查询，主要是不知道是否支持一对多的结果映射类似mybatis里的resultmap

**Reply by zhuxiujia** [reply]
> 我知道怎么写这个查询，主要是不知道是否支持一对多的结果映射类似mybatis里的resultmap

不支持mybatis的resultmap 功能，不过你可以用pysql! 或者 html_sql!写好关联查询后，用rbs::Value （返回的是Vec<Map>）作为返回结果
